### PR TITLE
API reference multi pages for v1.24

### PR DESCRIPTION
**Pull request by feloy**
Update of API Reference multi-pages (https://kubernetes.io/docs/reference/kubernetes-api/) for v1.24

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks! Some feedback / questions. **It's fine to merge even if the answers aren't clear.**

File: content/en/docs/reference/kubernetes-api/cluster-resources/node-v1.md [hunk 1]
```diff
@@ -121,10 +121,7 @@ NodeSpec describes the attributes that a node is created with.
 
     Required. The effect of the taint on pods that do not tolerate the taint. Valid effects are NoSchedule, PreferNoSchedule and NoExecute.
     
-    Possible enum values:
-     - `"NoExecute"` Evict any already-running pods that do not tolerate the taint. Currently enforced by NodeController.
-     - `"NoSchedule"` Do not allow new pods to schedule onto the node unless they tolerate the taint, but allow all pods submitted to Kubelet without going through the scheduler to start, and allow all already-running pods to continue running. Enforced by the scheduler.
-     - `"PreferNoSchedule"` Like TaintEffectNoSchedule, but the scheduler tries not to schedule new pods onto the node, rather than prohibiting new pods from scheduling onto the node entirely. Enforced by the scheduler.
```
[the remark this answers is not in the document]
Comment (feloy) [reply content/en/docs/reference/kubernetes-api/cluster-resources/node-v1.md:127]: I think these changes come from this PR: https://github.com/kubernetes/kubernetes/pull/109178

File: content/en/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 2]
```diff
@@ -114,9 +107,7 @@ NamespaceStatus is information about the current status of a Namespace.
 
   Phase is the current lifecycle phase of the namespace. More info: https://kubernetes.io/docs/tasks/administer-cluster/namespaces/
   
-  Possible enum values:
-   - `"Active"` means the namespace is available for use in the system
-   - `"Terminating"` means the namespace is undergoing graceful termination
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:119]: OK, I've filed https://github.com/kubernetes/kubernetes/issues/109463

File: content/en/docs/reference/kubernetes-api/cluster-resources/node-v1.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/reference/kubernetes-api/cluster-resources/node-v1.md:127]: OK, I've filed https://github.com/kubernetes/kubernetes/issues/109463

**Comment by nate-double-u** [comment]
Thanks a lot for his @feloy!

/lgtm

**Pull request merged**